.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```

Comment:
Could we also ask them here is they had a chance to review existing feature requests (link to issues with a feature request tag) and rather upvote/comment as opposed to creating a similar request?

Reply:
Yes I will link them to the the enhancement label in this repo.